src/core/include/openvino/core/parallel.hpp

```diff
@@ -48,6 +53,15 @@
 #    include "tbb/task_arena.h"
 #    include "tbb/task_scheduler_observer.h"
 
+static TBB_PARTITIONOR tbb_partitioner = TBB_STATIC;
```

Comment:
Different model can use different TBB partitioner to run inference in parallel. 

Please change this shared configuration to model related or stream executor related configuration.

Reply:
Updated.